src/modules/authz/hooks.js

```diff
@@ -81,35 +81,49 @@ export const useManagerPermissions = () => {
         hasAttempted: hasAttemptedManageProjectsDatasetsPermissions,
     } = useCanManageAtLeastOneProjectOrDataset();
 
-    const canViewDropBox = permissions.includes(viewDropBox);
-    const canIngest = permissions.includes(ingestData) || permissions.includes(ingestReferenceMaterial);
-    const canAnalyzeData = permissions.includes(analyzeData);
-    const canExportData = permissions.includes(exportData);
-    const canQueryData = permissions.includes(queryData);
-    const canViewRuns = permissions.includes(viewRuns);
-    const canViewPermissions = permissions.includes(viewPermissions);
-
-    return {
-        permissions: {
-            canManageProjectsDatasets,
-            canViewDropBox,
-            canIngest,
-            canAnalyzeData,
-            canExportData,
-            canQueryData,
-            canViewRuns,
-            canViewPermissions,
-            canManageAnything: (
-                canManageProjectsDatasets ||
-                canViewDropBox ||
-                canIngest ||
-                canAnalyzeData ||
-                canExportData ||
-                canViewRuns ||
-                canViewPermissions
-            ),
-        },
-        isFetching: isFetchingPermissions || isFetchingManageProjectsDatasetsPermissions,
-        hasAttempted: hasAttemptedPermissions && hasAttemptedManageProjectsDatasetsPermissions,
-    };
+    return useMemo(() => {
+        const canViewDropBox = permissions.includes(viewDropBox);
+        const canIngest = permissions.includes(ingestData) || permissions.includes(ingestReferenceMaterial);
+        const canAnalyzeData = permissions.includes(analyzeData);
+        const canExportData = permissions.includes(exportData);
+        const canQueryData = permissions.includes(queryData);
```

Comment:
Should `canQueryData` be a part of the `canManageAnything` permission? I get that it is not a "manager" permission per-se, but it is used in the manager to enable/disable the DRS tab, and `canManageAnything` exposes the data manager.

Reply:
the problem with it being part of `canManageAnything` is that then users which can just query data (i.e., most users) can then see the manager tab... maybe in the future we can have an explicit permission for seeing the manager tab that's separate, but as long as it's derived from permissions it makes sense to leave it out, I think

Reply:
Yeah I agree with you. That or we can have a permission for DRS querying and remove canQueryData from the manager permissions.